inductiva/tasks/task.py

```diff
@@ -153,14 +154,14 @@ def set_output_class(self, output_class):
         """Set the output class of the task."""
         self._output_class = output_class
 
-    def get_output(self, output_dir=None):
+    def get_output(self, output_dir: Optional[types.Path] = None):
         """Get the output of the task.
```

Comment:
Do you want to add something here about the type of output?
Also, this is a blocking task right?

Reply:
Yes, it is a blocking call, I've added info about that to the function. I've done  also the type annotations where it was missing.